benchbuild/project.py

```diff
@@ -308,7 +304,7 @@ def prepare(self):
     def compile(self):
         """Compile the project."""
 
-    def download(self, version=None):
+    def download(self, version: str = None):
```

Comment:
R0201 (no-self-use): Method could be a function  <!-- Automatically posted by Lintly -->